database/artix7/settings.sh

```diff
@@ -1 +1,25 @@
-source $(dirname ${BASH_SOURCE[0]})/../../settings/artix7.sh
```

Comment:
Whats with the settings.sh on this PR?

Reply:
I "totally" didn't _accidentally_ commit that :-P

Fixed.